<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>625</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>625</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>625</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ONDA | RÃ©initialisation du mot de passe</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* --- Global Style --- */
QMainWindow {
	background-color: #f3f4f6; /* Light gray background */
	font-family: Inter, sans-serif;
}

/* --- Main Frame Style --- */
QFrame#resetFrame {
	background-color: white;
	border: 1px solid #e5e7eb;
	border-radius: 12px;
}

/* --- Title and Labels Style --- */
QLabel#titleLabel {
    font-size: 20px;
    font-weight: bold;
    color: #1f2937;
    qproperty-alignment: 'AlignCenter';
}

QLabel {
	font-size: 14px;
	font-weight: bold;
	color: #1f2937;
}

/* --- Input Field Style --- */
QLineEdit {
	padding: 12px;
	border: 1px solid #d1d5db;
	border-radius: 8px;
	font-size: 14px;
	background-color: #ffffff;
}

QLineEdit:focus {
	border: 2px solid #2563eb; /* Blue border on focus */
}

/* --- Button Style --- */
QPushButton {
	background-color: #2563eb; /* Blue */
	color: white;
	padding: 12px;
	border: none;
	border-radius: 8px;
	font-size: 14px;
	font-weight: bold;
}

QPushButton:hover {
	background-color: #1d4ed8; /* Darker blue on hover */
}

/* --- 'Back to Login' Button Style --- */
QPushButton#backToLoginButton {
	border: none;
	font-size: 13px;
	background-color: #FFFFFF;
	color: #6b7281; /* Gray */
    qproperty-alignment: 'AlignCenter';
    font-weight: normal; /* Override bold */
}

QPushButton#backToLoginButton:hover {
	color: #1d4ed8; /* Blue on hover */
}

/* --- 'ONDA' Logo Text Style --- */
QLabel#logoText {
	font-size: 28px;
	font-weight: 800; /* Extra bold */
	color: #1e3a8a; /* Dark blue */
}

/* Make the stacked widget background transparent */
QStackedWidget, QStackedWidget &gt; QWidget {
    background-color: transparent;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <spacer name="verticalSpacer_6">
      <property name="orientation">
       <enum>Qt::Orientation::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Policy::Maximum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QFrame" name="frame">
      <property name="maximumSize">
       <size>
        <width>480</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">width: 1120px; 
height: 74px; 
background-color: #991B1B;
border-radius: 10px;</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::Shape::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Shadow::Plain</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="minimumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../../assets/icons.qrc">:/icons/icons/exclamation-circle.svg</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-size: 16px; 
font-weight: 500; 
color: #FFFFFF;</string>
         </property>
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainContent" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QFrame" name="resetFrame">
           <property name="minimumSize">
            <size>
             <width>400</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>400</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>20</number>
            </property>
            <property name="leftMargin">
             <number>30</number>
            </property>
            <property name="topMargin">
             <number>30</number>
            </property>
            <property name="rightMargin">
             <number>30</number>
            </property>
            <property name="bottomMargin">
             <number>30</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="logoLayout">
              <property name="spacing">
               <number>12</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>10</number>
              </property>
              <item>
               <spacer name="logoHorizontalSpacer_Left">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLabel" name="logoImage">
                <property name="minimumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>48</width>
                  <height>48</height>
                 </size>
                </property>
                <property name="styleSheet">
                 <string notr="true">background-color: #DBEAFE; border: 2px solid #1E3A8A; border-radius: 8px;</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="alignment">
                 <set>Qt::AlignmentFlag::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="logoText">
                <property name="text">
                 <string>ONDA</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="logoHorizontalSpacer_Right">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QLabel" name="titleLabel">
              <property name="text">
               <string>RÃ©initialiser le mot de passe</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QStackedWidget" name="stackedWidget">
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="page_verification">
               <layout class="QVBoxLayout" name="verticalLayout_3">
                <property name="spacing">
                 <number>15</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_username">
                  <property name="text">
                   <string>Nom d'utilisateur</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="usernameLineEdit">
                  <property name="placeholderText">
                   <string>Entrez votre nom d'utilisateur</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_email">
                  <property name="text">
                   <string>Adresse e-mail</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="emailLineEdit">
                  <property name="placeholderText">
                   <string>Entrez votre adresse e-mail</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Orientation::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Policy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>5</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="verifyButton">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>Consulter</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="page_new_password">
               <layout class="QVBoxLayout" name="verticalLayout_4">
                <property name="spacing">
                 <number>15</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="label_new_password">
                  <property name="text">
                   <string>Nouveau mot de passe</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="newPasswordLineEdit">
                  <property name="echoMode">
                   <enum>QLineEdit::EchoMode::Password</enum>
                  </property>
                  <property name="placeholderText">
                   <string>Entrez le nouveau mot de passe</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_confirm_password">
                  <property name="text">
                   <string>Confirmer le mot de passe</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="confirmPasswordLineEdit">
                  <property name="echoMode">
                   <enum>QLineEdit::EchoMode::Password</enum>
                  </property>
                  <property name="placeholderText">
                   <string>Confirmez le nouveau mot de passe</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Orientation::Vertical</enum>
                  </property>
                  <property name="sizeType">
                   <enum>QSizePolicy::Policy::Fixed</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>5</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="resetButton">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>45</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="text">
                   <string>RÃ©initialiser le mot de passe</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_5">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Policy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>10</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="backToLoginButton">
              <property name="cursor">
               <cursorShape>PointingHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>Retour Ã  la connexion</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../assets/icons.qrc"/>
 </resources>
 <connections/>
</ui>
